rfcs/0016-permissionless-assets-for-ibc-and-cw.md

```diff
@@ -0,0 +1,77 @@
+# Overview
+
+To transfer any asset from any chain connected via Centauri to DotSama, this asset needs to be defined in Picasso or Composable assets registry. This operation needs to be permissionless to allow free flow of assets between chains. Moreover, permissionless asset creation needs to support CosmWasm contracts which have functionality of creating, minting and burning tokens. There 2 types of permissionless assets: mintable assets(needed for cosmwasm) and nonmintable(for bridging). This document is an approach how to design permissionless assets and avoid potential risks.
+
+## What we currently have
+
+Currently we support asset creation in AssetsRegistry pallet via Root(Governance). During the permissioned asset creation process, user needs to specify asset's `protocol_id, nonce, name, symbol, decimals, location, ED, ratio`. 
+
+## What we want to achieve
+
+Permissionless assets need to follow the same structure as Permissioned assets and be part of Assets Registry. We want new assets to abide and not break interfaces we have via AssetsRegistry pallet and AssetsTransactorRouter pallet. Permissionless assets will also be modifiable via Root. For permissionless assets' creation a separate pallet is used to become a wrapper for AssetsRegistry: `permissionless_assets`. To create a new asset, user will need to pay registration fee in PICA to register a new asset.
```

Comment:
why permissionless assets are separate pallet? would wable to create assets which do not have metadata like symbol?

Reply:
we have router as wrapper? what for is router? there are 2 tasks about router. what for it is? how it will interact with these new assets?

Reply:
it is a separate pallet to store nonce and fix protocol id. Symbol and names are options. 

I dont understand the second question. This pallet will be a gateway for users to create assets that then will be manipulated the same way they are manipulated right now. AssetsTransactorRouter stays unchanged. This pallet will also allow CosmWasm contracts to mint tokens for their needs.